Podfile.lock

```diff
@@ -1,9 +1,9 @@
 PODS:
   - IP-UIKit-Wisdom (0.0.10)
-  - RxCocoa (4.0.0):
+  - RxCocoa (4.1.2):
     - RxSwift (~> 4.0)
-  - RxSwift (4.0.0)
-  - RxTest (4.0.0):
+  - RxSwift (4.1.2)
```

Comment:
What are our criteria for updating the `Podfile.lock`? Seems like this could have gone in without these changes.

Reply:
You're right. A version update shouldn't be part of this PR